build.gradle

```diff
@@ -20,6 +20,7 @@ ext.repos = [
                         'vlingo-auth': [version: '0.8.8'],
                       'vlingo-symbio': [version: '0.8.8'],
              'vlingo-symbio-dynamodb': [version: '0.8.8'],
+         'vlingo-symbio-foundationdb': [version: '0.8.8'],
```

Comment:
The vlingo-symbio-foundationdb repo has been purposely omitted from the build because it's difficult to get a server running on some developer machines. Also, it is experimental only, not considered officially released. Is there a reason that you need to have it built?

Reply:
Ah, no. It was included in the parent pom but missing after checkout, so I thought it was the checkout script that was off.